src/components/SideNav/SideNav.js

```diff
@@ -129,12 +128,9 @@ const NestedNav = ({
               key={index}
               items={el.items}
               title={el.title}
-              relativePath={el.parent.relativePath}
+              relativePath={el.parent && el.parent.relativePath}
```

Comment:
Nit on naming: `el` to me sounds like a DOM node/ref, maybe `subnavItem` instead? Not something I'd block the PR on though.